cmds/db-manager/main.go

```diff
@@ -0,0 +1,229 @@
+// Bootstrap script for Database deployment and migration
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"math"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database/cockroachdb"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	dssCockroach "github.com/interuss/dss/pkg/cockroach"
+	"go.uber.org/zap"
+	"golang.org/x/mod/semver"
+)
+
+// MyMigrate is an alias for extending migrate.Migrate
+type MyMigrate struct {
+	*migrate.Migrate
+	postgresURI string
+}
+
+// Direction is an alias for int indicating the direction and steps of migration
+type Direction int
+
+func (d Direction) String() string {
+	if d > 0 {
+		return "Up"
+	} else if d < 0 {
+		return "Down"
+	}
+	return "No Change"
+}
+
+var (
+	path      = flag.String("schemas_dir", "", "path to db migration files directory")
```

Comment:
Let's be clear about the behavior; since we don't yet have a README for this tool and process, how about: "path to db migration files directory.  the migrations found there will be applied to the database whose name matches the folder name."